.github/workflows/scripts/bump-versions.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# This script takes either current beta version, current stable version, or both,
+# and optionally next beta version and next stable version, and updates the version
+# in the specified files. If next version is not provided, it will infer the next 
+# semantic version (e.g. v0.110.0 -> v0.111.0 or v1.16.0 -> v1.17.0) based on the 
+# current version(s) passed.
+
+# Function to display usage
+usage() {
+  echo "Usage: $0 --current_beta <current_beta> [--current_stable <current_stable>] [--next_beta <next_beta>] [--next_stable <next_stable>]"
```

Comment:
This is a nit, but words in option names should probably be separated with a dash `-` and not an underscore `_`.
```suggestion
  echo "Usage: $0 --current-beta <current_beta> [--current-stable <current_stable>] [--next-beta <next_beta>] [--next-stable <next_stable>]"
```

Reply:
fixed, thanks